lib/ASRouter.jsm

```diff
@@ -1644,10 +1644,8 @@ class _ASRouter {
         "webextension-install-notify"
       );
       this.messageChannel.sendAsyncMessage(OUTGOING_MESSAGE_NAME, {
-        type: "CLEAR_MESSAGE",
-        data: { id: "RETURN_TO_AMO_1" },
+        type: "CLEAR_INTERRUPT",
       });
-      this.blockMessageById("RETURN_TO_AMO_1");
```

Comment:
Removing the block changes the behavior outside of the current page. Is that desired? I guess a subsequent opening of about:welcome is not common, but does this has other effects?

Is there a way to SET_MESSAGE something that would render the desired triplets to replace the CLEAR_MESSAGE ? This would avoid exposing interrupt-specific messages